src/containers/disputes/components/disputes-table/index.js

```diff
@@ -35,7 +35,17 @@ const columns = [
 ]
 
 const DisputesTable = ({ disputes }) => (
-  <Table columns={columns} loading={disputes.loading} data={disputes.data} />
+  <Table
+    columns={columns}
+    loading={disputes.loading}
+    data={disputes.data}
+    defaultSorted={[
```

Comment:
Is the array of disputes fetched from `0...disputes.length` (oldest...newest)?

Reply:
Yeah it is. So it comes in ordered by disputeId, which isn't necessarily oldest to newest with appeals